SIMDynPoroElasticity.h

```diff
@@ -0,0 +1,105 @@
+// $Id$
```

Comment:
Det kunne den vært, men jeg la den ikke der fordi denne kun brukes av PoroElasticity-appen, og ikke av FractureElasticity slik som de andre i PoroElastic-katalogen. Denne er ikke relevant for FractureElasticity, da den har sin egen driver-klasse som tilsvarer denne, nemlig SIMDynElasticity.

La gå, da er selvsagt brief doxyen feil her; denne er ikke for fracture - kun dynamisk poroelasticity.